Catalog/library_criteria.md

```diff
@@ -1,46 +1,43 @@
 1. Organizational recommendations facilitate community.
 
-Introduction
--------------
+## Introduction
 
 Open-source modeling ensures that model results are reproducible, that experts around the world can collaborate to make models better, and that users can explore the parameter space for themselves. These outcomes systematically improve public policy decisionmaking.
 
 We have developed the criteria in this document to facilitate the growth of an open source public policy modeling ecosystem.
 
-Criteria
----------
-PSL establishes three kinds of project Criteria. A project are required to (`MUST`) conform to `Acceptance Criteria` to be accepted in PSL. Projects are recommended to (`SHOULD`) or optionally (`MAY`) conform to various `Community Criteria` and `Interoperability Criteria`.
+## Criteria
 
-Acceptance Criteria for Transparency and Quality
---------------------------------------------
+### Mandatory Criteria for Transparency and Quality
 
 1. Models MUST be released under an OSI-approved open source license or the Creative Commons Public Domain Dedication (CC0).
 1. Data MUST be publicly available, unless release is restricted by a third party.
-1. For any data that SHOULD not be disclosed, provided MUST be:
+1. For any data that should not be disclosed, the project MUST provide:
 	- A complete descriptive list of all data variables;
 	- Descriptive statistics for all data variables for such data (including averages, standard deviations, number of observations, and correlations to other variables), to the extent that the descriptive statistics do not violate the rule against disclosure;
 	- Contact information for the individual or entity who has unrestricted access to the data.
-1. Projects MUST have unit tests and SHOULD report code coverage.
+1. Projects MUST have unit tests.
 1. At least one test MUST generate key outputs from source materials, the test MUST be run with every new version, and the outputs of the test MUST be checked into the repository.
 1. Projects MUST report names and contact information for at least one maintainer.
 1. Projects MUST have a suggested citation.
 1. Projects MUST have a project overview.
 1. Projects MUST have installation directions.
 1. Project MUST be mirrored in the same GitHub organization as PSL, and therefore they MUST be under version control.
-1. Projects MUST use a consistent versioning scheme, which SHOULD be [semantic versioning](https://semver.org/). If projects want to use the PSL Package-Builder tool to distribute packages via the Anaconda Cloud PSLmodels channel, there are [additional MUST criteria](https://github.com/PSLmodels/Package-Builder#using-package-builders-pbrelease-tool).
+1. Projects MUST use a consistent versioning scheme. If projects want to use the PSL Package-Builder tool to distribute packages via the Anaconda Cloud PSLmodels channel, there are [additional MUST criteria](https://github.com/PSLmodels/Package-Builder#using-package-builders-pbrelease-tool).
```

Comment:
Is Package-Builder still relevant?

Reply:
I do not think it is worth mentioning Package-Builder here. cc @MattHJensen